stable/anchore-engine/templates/engine_configmap.yaml

```diff
@@ -256,4 +256,11 @@ data:
         ssl_cert: "/home/anchore/certs/{{- .Values.anchoreGlobal.internalServicesSsl.certSecretCertName }}"
         ssl_key: "/home/anchore/certs/{{- .Values.anchoreGlobal.internalServicesSsl.certSecretKeyName }}"
         {{- end }}
-
+    runtime_inventory:
+      kubernetes:
+        report_anchore_cluster:
```

Comment:
These config values can just use the defaults in the values.yaml file. In the config I used the default function for "hidden" config values. I want them toggle-able, but they should only be toggled when you know exactly what you're doing, so I don't put them in the values file to keep people from changing it willy nilly. This is not considered helm best practice, but there are things that you only want people to change when you explicitly tell them to.

Reply:
Alternatively, we could stop defaulting configs and put them all into the values file like helm best practices say. We'd just want to comment them heavily so people know the implications.

Reply:
@Btodhunter Got it, I made the update, does it look right now?

Reply:
Yeah that looks great!